src/app/orders/page.tsx

```diff
@@ -1,69 +1,191 @@
-          {/* <p :class=fontColor>{{$t('label.profit')}}: <span>{{details.profit}} {{details.base}}</span></p> */}
+
+        <div className="columns">
+          <div className="column">
+            <p>single_purchase_amount: <span>{strategy.base_limit} {strategy.base}</span></p>
+            <p>total_purchased: <span>{strategy.base_total} {strategy.base}</span></p>
+            <p className={strategy.profit_percentage >= 0 ? 'has-text-success' : 'has-text-danger'}>current_value: <span>{strategy.price_total} {strategy.base}</span></p>
+          </div>
+          <div className="column">
+            <p>creation_time: <span>{strategy.createdAt}</span></p>
+            <p>quantity_bought: <span>{strategy.quote_total} {strategy.quote}</span></p>
+            <p className={strategy.profit_percentage >= 0 ? 'has-text-success' : 'has-text-danger'}>profit: <span>{strategy.profit} {strategy.base}</span></p>
+          </div>
+          <div className="column">
+            <p>stop_profit_rate: <span>{strategy.stop_profit_percentage||0} %</span></p>
+            <p>current_price: <span>{strategy.price_usd} {strategy.base}</span></p>
+            <p className={strategy.profit_percentage >= 0 ? 'has-text-success' : 'has-text-danger'}>profit_rate: <span>{strategy.profit_percentage}%</span></p>
+          </div>
         </div>
-        <div className="column">
-          <p>'label.stop_profit_rate' <span>details.stop_profit_percentage || 0 %</span></p>
-          <p>'label.current_price' <span>details.price_usd details.base</span></p>
-          {/* <p :class=fontColor>{{$t('label.profit_rate')}}: <span>{{details.profit_percentage}}%</span></p> */}
+
+        <div className="tabs">
+          <ul>
+            <li className="is-active"><a>Order History</a></li>
+          </ul>
         </div>
-      </div>
 
-      <div className={orderList.length ? 'order-line' : 'isNone order-line'} id="line-container">
-      </div>
-      <div className="tabs">
-        <ul>
-          <li className="is-active"><a>'caption.investment_orders'</a></li>
-        </ul>
-      </div>
-      
-      <table className='table is-fullwidth is-striped'  style={{ minWidth: '800px', fontSize: '0.85rem' }}>
-        <thead>
-          <tr>
-            <th>Create At</th>
-            <th>Pair</th>
-            <th>Price</th>
-            <th>Quantity</th>
-            <th>Price</th>
-            <th>Amount</th>
-            <th>Ave Price</th>
-            <th>Total</th>
-          </tr>
-        </thead>
-        
-        <tbody>
-        </tbody>
-      </table>
-    
+        <div className='table-wrapper'>
+          {orders && orders.length > 0 ? (
+            <OrderList orders={orders} />
+          ) : (
+            <Image src={no_record} alt='No records found' />
+          )}
+        </div>
       </div>
     </section>
   </div>
   );
 };
 
-export default orderList;
\ No newline at end of file
+export default StrategyDetails;
```

Comment:
export default OrderList?

Reply:
export StrategyDetails,because the orderlist is a part of strategy details